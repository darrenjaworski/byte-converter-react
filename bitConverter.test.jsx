import React from "react";
import renderer from "react-test-renderer";

import ByteConverter from "./index";

it("renders correctly", () => {
  const tree = renderer.create(<ByteConverter>1024</ByteConverter>).toJSON();
  expect(tree).toMatchSnapshot();
});
