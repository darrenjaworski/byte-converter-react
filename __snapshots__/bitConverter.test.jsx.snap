// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `"0.0009765625"`;
